contracts/FindMarket.cdc

```diff
@@ -1077,14 +1194,9 @@ pub contract FindMarket {
 			self.capability = nil
 		}
 
-		pub fun setMarketOption(name: String, cut: MetadataViews.Royalty?, rules: [TenantRule]) {
+		pub fun setMarketOption(saleItem: TenantSaleItem) {
```

Comment:
SetMarketOption now takes the saleItem struct. 
The tenant rules and cut is baked in the struct. 
It will be a little bit more visible from the transaction we ran on what we enabled